tencentcloud/resource_tc_kubernetes_cluster.go

```diff
@@ -726,10 +747,24 @@ func tkeGetCvmRunInstancesPara(dMap map[string]interface{}, meta interface{},
 	if v, ok := dMap["user_data"]; ok {
 		request.UserData = stringToPointer(v.(string))
 	}
-
-	chargeType := INSTANCE_CHARGE_TYPE_POSTPAID
-	request.InstanceChargeType = &chargeType
-
+	if v, ok := dMap["instance_charge_type"]; ok {
+		instanceChargeType := v.(string)
+		request.InstanceChargeType = &instanceChargeType
+		if instanceChargeType == CVM_CHARGE_TYPE_PREPAID {
+			request.InstanceChargePrepaid = &cvm.InstanceChargePrepaid{}
+			if period, ok := dMap["instance_charge_type_prepaid_period"]; ok {
+				periodInt64 := int64(period.(int))
+				request.InstanceChargePrepaid.Period = &periodInt64
+			} else {
+				errRet = fmt.Errorf("instance charge type prepaid period can not be empty when charge type is %s",
+					instanceChargeType)
+				return
+			}
+			if renewFlag, ok := dMap["instance_charge_type_prepaid_renew_flag"]; ok {
+				request.InstanceChargePrepaid.RenewFlag = stringToPointer(renewFlag.(string))
+			}
+		}
+	}
```

Comment:
1. It would be nice to have a unit test in `resource_tc_kubernetes_cluster_test.go`

Reply:
An idiotic question, how can I run the test locally? I can't find the `test.sh` file mentioned in readme. I tried `make testacc TEST=./tencentcloud` but it stops after running a few tests. And it looks like the test only can be executed with ap-guangzhou? (My default region is ap-shanghai and the az verification failed.)

Reply:
# How to test

## Setup SECRET_ID and SECRET_KEY

export TENCENTCLOUD_SECRET_ID=EXAMPLE
export TENCENTCLOUD_SECRET_KEY=EXAMPLE

## Write a test

Write test to `resource_tc_kubernetes_cluster_test.go`

## Run test

make testacc TESTARGS=" -run=^TestName$"

**However, it is ok not to test `PREPAY` tests, since it cann't delete immediately after test.**